continue-reading.html

```diff
@@ -1,13 +1,171 @@
+          libero quisquam, aut quis voluptatibus eos architecto ipsa maxime corrupti possimus? Eum similique quos
+          commodi reiciendis laboriosam beatae obcaecati fuga corporis provident in harum aliquid veritatis ducimus
+          laudantium odio quisquam, voluptatem, neque expedita facilis. Deserunt, iure in eligendi expedita culpa quo
+          fugiat non dolores, odio quas quam doloremque, nemo sequi dignissimos perferendis magni ipsa labore aliquid.
+
+        </p>
+        <a href="./continue-reading.html"><button id="main-featured-button"><img src="/images/noun_Book_121817.svg"> <p>continue reading</p></button></a>
+      </div>
+      <div class="h-break"></div>
+      <div id="feature-article-two" class="article-container">
+        <h2 class="header">article 2</h2>
+        <!--  -->
+        <img />
+        <p>
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Pariatur autem consectetur nulla at unde sit culpa
+          veritatis, voluptate, earum molestias libero, delectus error? Quisquam enim iste in, eligendi, voluptate, a
+          eum pariatur soluta totam ut dolore. Iure veniam unde quae ab assumenda distinctio culpa nulla vel quidem nam
+          quibusdam at quo dolores deserunt fugiat placeat, laboriosam voluptate non vitae odio minima saepe? Ab quasi
+          eaque nisi fuga eligendi autem ratione deserunt error optio, reprehenderit exercitationem maxime a deleniti
+          doloribus quia consequatur excepturi, eveniet amet minima debitis voluptatem. Vitae voluptate fugiat cumque
+          recusandae ducimus reprehenderit neque minus iusto rem omnis excepturi enim, amet quis! Sapiente maxime
+          molestiae optio voluptatibus? Est accusantium placeat nihil, ea reprehenderit dignissimos ipsa beatae
+          perferendis excepturi ipsum.
+
+          Lorem ipsum dolor sit amet consectetur adipisicing elit. Suscipit, sint? Quam nam temporibus a, nostrum,
+          possimus at inventore commodi ex, dolor sequi ipsum odio saepe nobis. Vero, possimus necessitatibus est omnis
+          repellendus aliquam quasi ducimus unde fugiat dignissimos, earum voluptates dolores et dolor debitis quo
+          facere ullam, architecto hic! Aut sapiente sunt nemo, quas rem, blanditiis ea fuga ratione quis exercitationem
+          eveniet unde harum minima quod iusto ab eum cumque? Corporis quae saepe ipsa minima laboriosam ut neque amet
+          dolorem iure reiciendis fuga nobis deleniti perspiciatis odio, veniam necessitatibus suscipit hic vitae eum id
+          architecto at beatae! Nulla rerum tempora, dolorum nemo libero at temporibus voluptate sequi rem quis
+          laboriosam voluptas qui. Ad ut hic numquam, voluptatem asperiores voluptas sed!
+
+        </p>
+        <a href="./continue-reading.html"><button id="main-featured-button"><img src="/images/noun_Book_121817.svg"> <p>continue reading</p></button></a>

```

Comment:
adding the css attribute to your anchor of `text-decoration: none` will remove the underline from your continue reading buttons, or if you wanted to keep the line there I believe you could set the color to black on the anchor and it would make the underline be black